txbits/public/js/vendor/zxcvbn.js

```diff
@@ -15,7 +15,7 @@ la=function(b){var a;a=b.token.charAt(0);a="a"===a||"1"===a?1:a.match(/\d/)?n(10
 c-1);for(e=k=1;1<=g?k<=g:k>=g;e=1<=g?++k:--k)f+=t(c-1,e-1)*h*Math.pow(d,e)}d=n(f);if(b.shifted_count){a=b.shifted_count;b=b.token.length-b.shifted_count;c=e=f=0;for(g=Math.min(a,b);0<=g?e<=g:e>=g;c=0<=g?++e:--e)f+=t(a+b,c);d+=n(f)}return d};da=function(b){b.base_entropy=n(b.rank);b.uppercase_entropy=ia(b);b.l33t_entropy=ha(b);return b.base_entropy+b.uppercase_entropy+b.l33t_entropy};$=/^[A-Z][^A-Z]+$/;Z=/^[^A-Z]+[A-Z]$/;Y=/^[^a-z]+$/;X=/^[^A-Z]+$/;ia=function(b){var a,d,c,e,f,g,h;f=b.token;if(f.match(X))return 0;
 e=[$,Z,Y];a=0;for(c=e.length;a<c;a++)if(b=e[a],f.match(b))return 1;a=function(){var a,b,c,e;c=f.split("");e=[];a=0;for(b=c.length;a<b;a++)d=c[a],d.match(/[A-Z]/)&&e.push(d);return e}().length;b=function(){var a,b,c,e;c=f.split("");e=[];a=0;for(b=c.length;a<b;a++)d=c[a],d.match(/[a-z]/)&&e.push(d);return e}().length;c=g=e=0;for(h=Math.min(a,b);0<=h?g<=h:g>=h;c=0<=h?++g:--g)e+=t(a+b,c);return n(e)};ha=function(b){var a,d,c,e,f,g,h,i,j,k;if(!b.l33t)return 0;f=0;j=b.sub;for(g in j){h=j[g];a=function(){var a,
 d,e,f;e=b.token.split("");f=[];a=0;for(d=e.length;a<d;a++)c=e[a],c===g&&f.push(c);return f}().length;d=function(){var a,d,e,f;e=b.token.split("");f=[];a=0;for(d=e.length;a<d;a++)c=e[a],c===h&&f.push(c);return f}().length;e=i=0;for(k=Math.min(d,a);0<=k?i<=k:i>=k;e=0<=k?++i:--i)f+=t(d+a,e)}return n(f)||1};C=function(b){var a,d,c,e,f,g,h,i;f=[!1,!1,!1,!1,!1];c=f[0];g=f[1];d=f[2];e=f[3];f=f[4];i=b.split("");b=0;for(h=i.length;b<h;b++)a=i[b],a=a.charCodeAt(0),48<=a&&57>=a?d=!0:65<=a&&90>=a?g=!0:97<=a&&
-122>=a?c=!0:127>=a?e=!0:f=!0;b=0;d&&(b+=10);g&&(b+=26);c&&(b+=26);e&&(b+=33);f&&(b+=100);return b};fa=function(b){return 60>b?"instant":3600>b?""+(1+Math.ceil(b/60))+" minutes":86400>b?""+(1+Math.ceil(b/3600))+" hours":2678400>b?""+(1+Math.ceil(b/86400))+" days":32140800>b?""+(1+Math.ceil(b/2678400))+" months":321408E4>b?""+(1+Math.ceil(b/32140800))+" years":"centuries"};var E={"!":["`~",null,null,"2@","qQ",null],'"':[";:","[{","]}",null,null,"/?"],"#":["2@",null,null,"4$","eE","wW"],$:["3#",null,
+122>=a?c=!0:127>=a?e=!0:f=!0;b=0;d&&(b+=10);g&&(b+=26);c&&(b+=26);e&&(b+=33);f&&(b+=100);return b};fa=function(b){return 60>b?Messages("zxvbn.instant"):3600>b?""+(1+Math.ceil(b/60))+" "+Messages("zxvbn.minutes"):86400>b?""+(1+Math.ceil(b/3600))+ " " +Messages("zxvbn.hours"):2678400>b?""+(1+Math.ceil(b/86400))+ " "+Messages("zxvbn.days"):32140800>b?""+(1+Math.ceil(b/2678400))+" "+Messages("zxvbn.months"):321408E4>b?""+(1+Math.ceil(b/32140800))+" "+Messages("zxvbn.years"):Messages("zxvbn.centuries")};var E={"!":["`~",null,null,"2@","qQ",null],'"':[";:","[{","]}",null,null,"/?"],"#":["2@",null,null,"4$","eE","wW"],$:["3#",null,
```

Comment:
Oh I see, I guess I read that wrong. Is there any way to do this without modifying the library file?

On February 5, 2016 2:39:53 AM PST, Yura <notifications@github.com> wrote:
>> @@ -15,7 +15,7 @@ la=function(b){var
>a;a=b.token.charAt(0);a="a"===a||"1"===a?1:a.match(/\d/)?n(10
>> 
>c-1);for(e=k=1;1<=g?k<=g:k>=g;e=1<=g?++k:--k)f+=t(c-1,e-1)*h*Math.pow(d,e)}d=n(f);if(b.shifted_count){a=b.shifted_count;b=b.token.length-b.shifted_count;c=e=f=0;for(g=Math.min(a,b);0<=g?e<=g:e>=g;c=0<=g?++e:--e)f+=t(a+b,c);d+=n(f)}return
>d};da=function(b){b.base_entropy=n(b.rank);b.uppercase_entropy=ia(b);b.l33t_entropy=ha(b);return
>b.base_entropy+b.uppercase_entropy+b.l33t_entropy};$=/^[A-Z][^A-Z]+$/;Z=/^[^A-Z]+[A-Z]$/;Y=/^[^a-z]+$/;X=/^[^A-Z]+$/;ia=function(b){var
>a,d,c,e,f,g,h;f=b.token;if(f.match(X))return 0;
>>  e=[$,Z,Y];a=0;for(c=e.length;a<c;a++)if(b=e[a],f.match(b))return
>1;a=function(){var
>a,b,c,e;c=f.split("");e=[];a=0;for(b=c.length;a<b;a++)d=c[a],d.match(/[A-Z]/)&&e.push(d);return
>e}().length;b=function(){var
>a,b,c,e;c=f.split("");e=[];a=0;for(b=c.length;a<b;a++)d=c[a],d.match(/[a-z]/)&&e.push(d);return
>e}().length;c=g=e=0;for(h=Math.min(a,b);0<=h?g<=h:g>=h;c=0<=h?++g:--g)e+=t(a+b,c);return
>n(e)};ha=function(b){var a,d,c,e,f,g,h,i,j,k;if(!b.l33t)return
>0;f=0;j=b.sub;for(g in j){h=j[g];a=function(){var a,
>> 
>d,e,f;e=b.token.split("");f=[];a=0;for(d=e.length;a<d;a++)c=e[a],c===g&&f.push(c);return
>f}().length;d=function(){var
>a,d,e,f;e=b.token.split("");f=[];a=0;for(d=e.length;a<d;a++)c=e[a],c===h&&f.push(c);return
>f}().length;e=i=0;for(k=Math.min(d,a);0<=k?i<=k:i>=k;e=0<=k?++i:--i)f+=t(d+a,e)}return
>n(f)||1};C=function(b){var
>a,d,c,e,f,g,h,i;f=[!1,!1,!1,!1,!1];c=f[0];g=f[1];d=f[2];e=f[3];f=f[4];i=b.split("");b=0;for(h=i.length;b<h;b++)a=i[b],a=a.charCodeAt(0),48<=a&&57>=a?d=!0:65<=a&&90>=a?g=!0:97<=a&&
>>
>-122>=a?c=!0:127>=a?e=!0:f=!0;b=0;d&&(b+=10);g&&(b+=26);c&&(b+=26);e&&(b+=33);f&&(b+=100);return
>b};fa=function(b){return 60>b?"instant":3600>b?""+(1+Math.ceil(b/60))+"
>minutes":86400>b?""+(1+Math.ceil(b/3600))+"
>hours":2678400>b?""+(1+Math.ceil(b/86400))+"
>days":32140800>b?""+(1+Math.ceil(b/2678400))+"
>months":321408E4>b?""+(1+Math.ceil(b/32140800))+"
>years":"centuries"};var
>E={"!":["`~",null,null,"2@","qQ",null],'"':[";:","[{","]}",null,null,"/?"],"#":["2@",null,null,"4$","eE","wW"],$:["3#",null,
>>
>+122>=a?c=!0:127>=a?e=!0:f=!0;b=0;d&&(b+=10);g&&(b+=26);c&&(b+=26);e&&(b+=33);f&&(b+=100);return
>b};fa=function(b){return
>60>b?Messages("zxvbn.instant"):3600>b?""+(1+Math.ceil(b/60))+"
>"+Messages("zxvbn.minutes"):86400>b?""+(1+Math.ceil(b/3600))+ " "
>+Messages("zxvbn.hours"):2678400>b?""+(1+Math.ceil(b/86400))+ "
>"+Messages("zxvbn.days"):32140800>b?""+(1+Math.ceil(b/2678400))+"
>"+Messages("zxvbn.months"):321408E4>b?""+(1+Math.ceil(b/32140800))+"
>"+Messages("zxvbn.years"):Messages("zxvbn.centuries")};var
>E={"!":["`~",null,null,"2@","qQ",null],'"':[";:","[{","]}",null,null,"/?"],"#":["2@",null,null,"4$","eE","wW"],$:["3#",null,
>
>I have translated not dictionaries, and message : minutes, hours ,
>days, months , years.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/txbits/txbits/pull/104/files#r52000223

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.